### PR TITLE
Added mci_transformation schema to SR

### PR DESCRIPTION
**Pull request by shahi645**
### What is the context of this PR?
New `mci_transformation` schema. Also made some tiny content changes for `rsi_transformation`

### How to review 
Does the spec match the schema?

### CODE REVIEW

**Review by nealedj — APPROVED** [review]

File: data/en/mci_transformation.json [hunk 1]
```diff
@@ -0,0 +1,952 @@
...
+                        "title": "Information you need",
+                        "content": [{
+                            "description": "You can select the dates of the period you are reporting for, if the given dates are not appropriate."
+                        }],
+                        "questions": [{
+                                "question": "Total Turnover excluding VAT",
+                                "content": [{
+                                        "description": "Include:",
+                                        "list": [
+                                            "commission and fees",
```
Comment (dwyeradam) [inline data/en/mci_transformation.json]: commission`s` and fees
Missing the s from the end of commissions.

File: data/en/mci_transformation.json [hunk 2]
```diff
@@ -0,0 +1,952 @@
...
+                    "title": "Total turnover excluding VAT",
+                    "questions": [{
+                        "id": "total-turnover-question",
+                        "title": "For the period {{format_conditional_date(answers['period-from'], metadata['ref_p_start_date'])}} to {{format_conditional_date(answers['period-to'], metadata['ref_p_end_date'])}}, how much was {{metadata['trad_as_or_ru_name']}}’s <em>total turnover excluding VAT</em>?",
+                        "type": "General",
+                        "guidance": {
+                            "content": [{
+                                    "title": "Include",
+                                    "list": [
+                                        "commission and fees",
```
Comment (dwyeradam) [inline data/en/mci_transformation.json]: commission`s` and fees

File: data/en/mci_transformation.json [hunk 3]
```diff
@@ -0,0 +1,952 @@
...
+                                            "toys and games",
+                                            "cycles and cycle accessories",
+                                            "sport and camping equipment",
+                                            "jewellery",
+                                            "silverware and plate, clocks and watches",
+                                            "household cleaning products and kitchen paper products",
+                                            "pets, pets’ requisites and pet foods",
+                                            "cut flowers, plants, seeds and other garden sundries",
+                                            "other new and second hand goods",
+                                            "Mobile phones"
```
Comment (dwyeradam) [inline data/en/mci_transformation.json]: `m`obile phones
lower case M

**Review by dwyeradam — CHANGES_REQUESTED** [review]
Couple of minor text changes which i'll do myself and push.

**Review by dwyeradam — APPROVED** [review]

**Pull request merged**